### ISSUE TITLE
Index out of bounds of array

### ISSUE THREAD

**Issue by blake-haas** [Adrianotiger/desktopPet#34]
With latest code:
getting an index out of bounds array on an animation frame it looks like:
Animation ID : 1
Animation Name: Walk
Animation Sequence: 2,3,

Could be on other animation id's as well. I just finally got this so many times to post this.

**Comment by Adrianotiger** [comment]
Hi Blake,
Do you start the application in debug mode (do you see the log window)?
I had this only when I close the debug window or when a new graphic driver is installed, probably because it removes the screens and the pet does not know where to render. 

I would like to reproduce it. Do you starts some applications in fullscreen?

**Comment by blake-haas** [comment]
No debug window. I believe this is also the portable build. It also
happened with the installer on the website which is why I downloaded source
and tried that.

I just have it run on startup. Whenever it happens I right click and add a
new sheep and they keep moving for a while until they lock up. It is on a
laptop that is docked and undocked but I've observed it happening when
doing nothing on screen.
